Atomic/rpmwriter.py

```diff
@@ -0,0 +1,369 @@
+import os
```

Comment:
Sounds good, I misunderstood your comment on the python3 failure.  Given that it would stlll be good to have some kind of shebang statement at the top of the file.  Perhaps:

#!/usr/bin/env python

But certainly not a comment to hold the merge for.